### DISCUSSION TITLE
🇩🇪 als Ausstattung

### DISCUSSION THREAD

**Discussion by Tramhendi** [marhei/TrainCompany-Data#280] [Ideas]
Es gibt ja nun einige Fahrzeuge, die nicht in Deutschland fahren dürfen und für die dementsprechend ein besonderer Hinweis angebracht werden musste, dass sie bspw. nur in der Schweiz verkehren können.
Wäre es da nicht sinnvoll, auch Deutschland als Länderausstattung einzuführen, um das konsistenter zu regeln?

(Oder wurde das evtl. schon überlegt, aber verworfen, weil es zu aufwändig wäre, das jetzt für alle deutschen Strecken und Fahrzeuge nachträglich zu ändern?)

**Comment by marhei** [comment]
Nein, nicht sinnvoll…
